deployment/clusterObjectModel/forward_compatibility.py

```diff
@@ -17,59 +17,76 @@
 
 
 
+def transform(old_model, old_key, new_model, new_key):
+    old_key_list = old_key.split('.')
+    new_key_list = new_key.split('.')
+    old_dict = old_model
+    new_dict = new_model
+    for key in old_key_list:
+        if key not in old_dict:
+            return
+        old_dict = old_dict[key]
+    for key in new_key_list:
```

Comment:
looks like you only need to iterate over `new_key_list[:-1]` and remove the check of `!= new_key_list[-1]`

Reply:
Done